python/plugins/MetaSearch/dialogs/manageconnectionsdialog.py

```diff
@@ -164,7 +166,7 @@ def load(self, items):
 
             # no dups detected or overwrite is allowed
             url = '/MetaSearch/%s/url' % conn_name
-            self.settings.setValue(url, csw.attrib.get('url'))
+            self.settings.setValue(url, catalog.attrib.get('url'))
```

Comment:
shouldn't this be setting the type value here too?

Reply:
fixed/pushed.